### ISSUE TITLE
Code of Conduct Plan of Action

### ISSUE THREAD

**Issue by tobie** [openjs-foundation/cross-project-council#1122]
This is main issue that centralization [all current CoC work](https://github.com/openjs-foundation/cross-project-council/labels/TOPIC-code-of-conduct)

- [ ] Pull request to **remove automation** (@joesepi)
  - [ ] Remove automation code and supporting files
  - [ ] Remove any reference to automation in the CoC and elsewhere in the repo
  - Closes #1064.
- [ ] Pull request to **update the structure, nomination, and process of the CoC Team** (@tobie)
  - See [notes taken during CoC working session](https://docs.google.com/document/d/1ChxS1TN0ga7U5vlEWHalmQDCOTFcuKJsjs55Gh0fKaE/edit).
  - [ ] Clarify which issues or existing PRs are related.
- [ ] **Restart nomination process** (following up on the existing open issue).
  - [ ] Clarify which issues or existing PRs are related.
- [ ] **Update CoC to Contributor Covenant 2.1** (@joesepi)
  - [ ] include preamble that clarifies that “project” is to be interpreted as representing the whole OpenJSF community.
  - [ ] Clarify which issues or existing PRs are related.
- [ ] **Move the CoC to a .github repo** (@ljharb)
  - [ ] Assess impact for OpenJSF projects that are not part of this org.

**Comment by tobie** [comment]
@tobie to look into next step of selection process.

**Comment by joesepi** [comment]
Meeting notes:
  * Big epic for code of conduct changes; waiting on CoC team selection process and then will revisit this to be sure we’ve checked all the boxes
